content/tutorials/manage-account-settings/offline-account-setup.md

```diff
@@ -0,0 +1,332 @@
+
+
+
+### {{n.next()}}. Fund the new address
+
+From an online machine, send enough XRP to the **account address** you noted in step 1. For more information, see [Creating Accounts](accounts.html#creating-accounts).
+
+**Tip:** For testing purposes, you can use the [Testnet Faucet](xrp-testnet-faucet.html) to get a new account with Test XRP, then use that account to fund the address you generated offline.
+
+
+
+### {{n.next()}}. Confirm account details
+
+When the transaction from the previous step is validated by consensus, your account has been created. From the online machine, you can confirm the status of the account with the [account_info method][]. Make sure the response contains `"validated": true` to confirm that this result is final.
+
+Take note of the sequence number of the account, in the `Sequence` field of the result's `account_data`. You need to know the sequence number to sign transactions from the account in future steps.
+
+If the [DeletableAccounts amendment](known-amendments.html#deletableaccounts) :not_enabled: is enabled, the `Sequence` number of a newly-funded account matches the [ledger index][] when it was funded. Otherwise, a newly funded account's `Sequence` number is always 1.
+
+<!-- MULTICODE_BLOCK_START -->
+
+_rippled Commandline_
+
+```sh
+$ ./rippled account_info rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn
+
+Loading: "/etc/opt/ripple/rippled.cfg"
+2019-Dec-11 01:06:21.728637950 HTTPClient:NFO Connecting to 127.0.0.1:5005
+{
+   "result" : {
+      "account_data" : {
+         "Account" : "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+         "Balance" : "5000000000000",
+         "Flags" : 0,
+         "LedgerEntryType" : "AccountRoot",
+         "OwnerCount" : 0,
+         "PreviousTxnID" : "00C5B713B11DA775C6F932D38CE162C16FA88B7269BAFC6FDF4C6ADB74419670",
+         "PreviousTxnLgrSeq" : 3,
+         "Sequence" : 1,
+         "index" : "13F1A95D7AAB7108D5CE7EEAF504B2894B8C674E6D68499076441C4837282BF8"
+      },
+      "ledger_current_index" : 4,
+      "status" : "success",
+      "validated" : false
+   }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+### {{n.next()}}. Enter the sequence number on the offline machine.
+
+Save the account's starting sequence number on the offline machine as its current sequence number. Whenever you sign a new transaction using the offline machine, you will use the current sequence number, then increase the current sequence number by 1.
+
+You can prepare several transactions in advance this way, then transfer the signed transactions to the online machine all at once and submit them. As long as each transaction is validly formed and pays a sufficient [transaction cost](transaction-cost.html), the XRP Ledger network should eventually include those transactions in validated ledgers, keeping the account's sequence number in the shared XRP Ledger in sync with the "current" sequence number you are tracking on the offline machine.
```

Comment:
In principal, if a transaction does not specify a `LastLedgerSequence` field, and no other transaction is submitted for that account with the same sequence number, that transaction could succeed at a completely arbitrary time in the future: days, months, even years. In practice, it's generally going to be less than 15 seconds, but with queueing on a busy network, it could be several minutes. However, because there are so many open-ended possibilities, most places in the documentation and the code leave the time frame intentionally vague, except where we emphasise using a `LastLedgerSequence`.

Reply:
Still, it's OK to provide some expectations for the typical case. Added:

> (Most transactions get a final, validated result within 15 seconds or less after being submitted to the network.)